io/disk/lvsetup.py.data/lvsetup.yaml

```diff
@@ -0,0 +1,6 @@
+scenario: !mux
+    loopback:
+        disk: None
+        lv_size: "10M"
+    real:
+        disk: /dev/sde
```

Comment:
I'd strongly suggest commenting this second one out to avoid people complaining about wiped drives...

```
# Uncomment this to use real drive (warning: it'll be erased!)
# real:
#    disk: /dev/sde

And just to be sure, you might actually use `/dev/vde` to avoid unnecessary corruption of main drive...
```